src/NuGet.Clients/NuGet.PackageManagement.UI/Design/PackageItemListViewSampleData.xaml

```diff
@@ -7,10 +7,5 @@
   xmlns:sys="clr-namespace:System;assembly=mscorlib"
   Type="{x:Type nuget:PackageItemViewModel}"
   >
-  <nuget:PackageItemViewModel
-    Id="TestPackage1"
-    />
-  <nuget:PackageItemViewModel
-    Id="TestPackage2"
-    />
+  <nuget:PackageItemViewModel />
```

Comment:
Question: why are we going from 2 to 1 instances of this view model class in the sample data? Is this related to our changes?

Reply:
Removed by mistake, maybe we need to fix this later, it's a SampleData.xml, because Id cannot be set, we need to pass a package model instead.